java/src/main/java/ai/rapids/cudf/ColumnFilterOptions.java

```diff
@@ -21,7 +21,6 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
-import java.util.function.Supplier;
```

Comment:
Nit: needs updated copyrights if we're going to modify this file.

Reply:
updated